### PR TITLE
cgp-0130

### PR DESCRIPTION
**Pull request by reneerrusso**

### CODE REVIEW

**Review by juanjgiraldoc — APPROVED** [review]

**Review by 0xzoz — APPROVED** [review]

File: CGPs/cgp-0130.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
+cgp: 0130
+title: Aligned Public Goods Bucket: Red Beard Ventures x CELO Accelerator
```
Comment (0xzoz) [inline CGPs/cgp-0130.md:3]: The second colon needs to be removed in title for the CI to pass please @reneerrusso. Here is an example

---
cgp: 129
title: RWA-focused Accelerator with Outlier Ventures
date-created: 2024-04-03
author: 'John Goldschmidt (@Johnlikes), Aron van Ammers (@aronvanammers)'
status: DRAFT
discussions-to: https://forum.celo.org/t/draft-celo-public-goods-grant-request-update-rwa-focused-accelerator-with-outlier-ventures/
governance-proposal-id:
date-executed:
---

**Review by 0xzoz — CHANGES_REQUESTED** [review]

File: CGPs/cgp-0130.md (hunk 1 above)
Comment (0xzoz) [reply CGPs/cgp-0130.md:3]: title: Aligned Public Goods Bucket: (<------this one) Red Beard Ventures x CELO Accelerator

**Comment by juanjgiraldoc** [comment]
@reneerrusso can you remove execution date pls

**Comment by juanjgiraldoc** [comment]
@reneerrusso  And also Governance Proposal Id this is only known when we submit the proposal OnChain

**Review by juanjgiraldoc — APPROVED** [review]

**Pull request merged**